pkgs/leak_tracker_testing/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: leak_tracker_testing
+version: 1.0.0
+description: Leak tracking code intended for usage in tests.
+repository: https://github.com/dart-lang/leak_tracker
+
+environment:
+  sdk: '>=3.0.0 <4.0.0'
+
+dependencies:
+  leak_tracker: ^7.0.4
```

Comment:
Same comment here re: referencing the repo version of `leak_tracker`.

Reply:
Ah, ok. I'm not familiar w/ why this is published, but given that it is, try:

for the pubspec:

```yaml
dependencies:
  leak_tracker: ^7.0.4
```

and create a file next to it called `pubspec_overrides.yaml`:

```yaml
dependency_overrides:
  leak_tracker:
    path: ../leak_tracker
```

Reply:
it worked inside the same file